packages/components/src/Chips/Chips.mdx

```diff
@@ -17,6 +18,14 @@ import { Text } from "@jobber/components/Text";
 
 # Chips
 
+<!-- DELETE ME: For testing purposes, delete before putting up for review -->
+
+### Add/dismiss selection playgound
```

Comment:
I can see a case where we may have a button to add another chip to the group. For example, could be used for adding tags to a page. 
![Screen Shot 2022-01-19 at 5 22 06 PM](https://user-images.githubusercontent.com/7876533/150224888-f9c17e19-5cba-491f-a8d6-382908930708.png)


I can also think of cases (e.g. compose message, send email/sms modal) where we don't have a button, but instead just show the input area where they would be typing in chips to add. Here is an example from Jobber Online:

![Screen Shot 2022-01-19 at 5 20 23 PM](https://user-images.githubusercontent.com/7876533/150224143-63ffbc6d-e28b-4229-a315-38a778a81952.png)

Reply:
With the send email/sms modal, adding a comma in between strings is also used to create a new chip. Makes sense as functionality if the input allows for any value

Reply:
![Screen Shot 2022-01-19 at 5 29 24 PM](https://user-images.githubusercontent.com/7876533/150228880-7cbbfb58-5733-46a9-a1c0-e592c010ab8d.png)

If you enter a term that has matches, wondering if it makes more sense to have the "Add da" option listed first instead of last. If I wanted to input a custom value, typing in the value and hitting enter would select the first match on the list rather than the custom value I entered.